app/controllers/campaign_pages_controller.rb

```diff
@@ -23,7 +23,7 @@ def create
     @campaign_page = CampaignPage.new(@page_params)
     @campaign_page.compile_html
     if @campaign_page.save
-      ChampaignQueue::SqsPusher.push(@campaign_page.as_json)
+      ChampaignQueue::SqsPusher.push({type: 'create', params: @campaign_page}.to_json)
```

Comment:
I'd keep `as_json` if working with ActiveRecord collections. I had `to_json` first but it littered the object with escape characters. `to_json` in the sign method works all right, but for some reason when working with AR collections, I needed to use both.